esmvaltool/recipes/recipe_hyint.yml

```diff
@@ -0,0 +1,85 @@
+# recipe_hyint.yml
+---
+
+documentation:
+  description: |
+    Recipe for the HyInt package by E. Arnone and J. von Hardenberg (ISAC-CNR)
+
+  authors:
+    - arno_en
+    - hard_jo
+
+  references:
+    - giorgi11jc
+    - giorgi14jgr
+
+  projects:
+    - c3s-magic
+
+datasets:
+  - {dataset: ACCESS1-0,      project: CMIP5, exp: [historical,rcp85], ensemble: r1i1p1, start_year: 2004, end_year: 2006}
+#  - {dataset: ACCESS1-3,      project: CMIP5, exp: rcp85, ensemble: r1i1p1, start_year: 2006, end_year: 2016}
+#  - {dataset: MPI-ESM-MR,      project: CMIP5, exp: rcp85, ensemble: r1i1p1, start_year: 2006, end_year: 2016}
+
+preprocessors:
+  preproc:
+    extract_levels: false
```

Comment:
There is no need to disable this preprocessor function, it is disabled by default

Reply:
updated